### PR TITLE
TCVP-2704 Fix Blank Fields in DCF

### PR DESCRIPTION
**Pull request by karahanb**
# Description

This PR includes the following proposed change(s):

- [TCVP-2704](https://justice.gov.bc.ca/jira/browse/TCVP-2704)
- Fixed the issue with displaying some of the fields including Room, Judicial Justice based on the returned data.

room-jj-data

hearing-duration

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.ts [hunk 1]
```diff
@@ -79,7 +79,8 @@ export class JJDisputeComponent implements OnInit {
   });
   courtAppearanceForm: FormGroup = this.formBuilder.group({
     appCd: [null],
-    room: [{ value: null, disabled: true }],
+    room: [null],
+    duration: [null],
```
Comment (Sharkww) [inline src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.ts:83]: Is `adjudicatorName` still using ?

File: src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.ts [hunk 2]
```diff
@@ -179,7 +180,6 @@ export class JJDisputeComponent implements OnInit {
         if (!this.lastUpdatedJJDispute.mostRecentCourtAppearance.jjSeized) this.lastUpdatedJJDispute.mostRecentCourtAppearance.jjSeized = 'N';
 
         // if (!this.isViewOnly) {
-        this.lastUpdatedJJDispute.mostRecentCourtAppearance.adjudicator = this.jjIDIR;
         this.lastUpdatedJJDispute.mostRecentCourtAppearance["adjudicatorName"] = this.jjName; // force to add property
```
Comment (Sharkww) [inline src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.ts:183]: Is `adjudicatorName` still using ?

**Comment by Sharkww** [comment]
And I think `Location of Offence` is still missing.

**Review by Sharkww — APPROVED** [review]

File: src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.ts (hunk 2 above)
Comment (karahanb) [reply src/frontend/staff-portal/src/app/components/jj-dispute-info/jj-dispute/jj-dispute.component.ts:183]: Yes, I think that is still used for the logged in JJ.

**Comment by karahanb** [comment]
> And is `Location of Offence` still missing ? Or it is just the problem of data ? If so, you can merge it.

I haven't changed anything related to `Location of Offence` because it was already displaying that correctly. The ones that missing the `Location of Offence` must be a data issue.

**Pull request merged**